--- nginx.conf.orig	2017-05-24 08:58:28.902144805 +0000
+++ nginx.conf	2017-05-24 11:32:39.390466022 +0000
@@ -29,11 +29,12 @@
 
     #keepalive_timeout  0;
     keepalive_timeout  65;
+    push_stream_shared_memory_size 32M;
 
     #gzip  on;
 
     server {
-        listen       80;
+        listen       8080;
         server_name  localhost;
 
         #charset koi8-r;
@@ -54,6 +55,32 @@
             root   html;
         }
 
+
+  location /pub {
+    push_stream_publisher admin;
+    push_stream_channels_path               $arg_id;
+    allow all;
+  }
+
+  location ~ /sub/(.*) {
+    push_stream_subscriber;
+    push_stream_channels_path                   $1;
+    push_stream_message_template                "{\"id\":~id~,\"channel\":\"~channel~\",\"text\":~text~, \"time\":\"~time~\", \"eventid\":\"~event-id~\"}";
+
+  }
+
+  location ~ /ws/(.*) {
+    push_stream_subscriber websocket;
+    push_stream_channels_path                   $1;
+    push_stream_message_template                "{\"id\":~id~,\"channel\":\"~channel~\",\"text\":~text~, \"time\":\"~time~\", \"eventid\":\"~event-id~\"}";
+
+  }
+  location ~ /ev/(.*) {
+    push_stream_subscriber                      eventsource;
+    push_stream_channels_path                   $1;
+    push_stream_message_template                "{\"id\":~id~,\"channel\":\"~channel~\",\"text\":~text~}";
+  }
+
         # proxy the PHP scripts to Apache listening on 127.0.0.1:80
         #
         #location ~ \.php$ {
